bpf/bpf_dbg.h

```diff
@@ -32,6 +32,20 @@ struct {
 
 enum bpf_func_id___x { BPF_FUNC_snprintf___x = 42 /* avoid zero */ };
 
+#ifdef BPF_DEBUG_TC
```

Comment:
I'm wondering whether we should add the following to the top of this file, just so that we can pass `-DBPF_DEBUG_TC` directly:

```
#ifdef BPF_DEBUG_TC
#define BPF_DEBUG
#endif 
```